index.js

```diff
@@ -33,9 +39,9 @@ module.exports = function(glob, options) {
 
   // make glob absolute
   if (rootDir && glob.charAt(0) === '/') {
-    glob = path.join(rootDir, glob);
-  } else if (!isAbsolute(glob)) {
-    glob = path.resolve(cwd, glob);
+    glob = join(rootDir, glob);
+  } else if (process.platform === 'win32' || !isAbsolute(glob)) {
```

Comment:
This can be avoided by using the `/test/123` test even on Windows but I don't know if that's how most users think.